net/NetUtil.cpp

```diff
@@ -197,10 +197,7 @@ HostEntry syncResolveDNS(const std::string& addressToCheck)
     std::shared_ptr<HostEntry> result;
 
     net::AsyncDNS::DNSThreadDumpStateFn dumpState = [addressToCheck]() -> std::string
-    {
-        std::string state = "syncResolveDNS: [" + addressToCheck + "]";
```

Comment:
Seems unlikely we'd need to look at `state` in the debugger, so made it implicit.